web_services/foxx/brightid/db.js

```diff
@@ -502,11 +526,10 @@ function appToDic(app) {
     id: app._key,
     name: app.name,
     context: app.context,
-    verification: getContext(app.context).verification,
+    verification: getContext(app.context) ? getContext(app.context).verification : null,
```

Comment:
We should load verification expression from app not from context.
Each app should have a verification expression that should be synced among nodes via the json file. The default value should be `BrightID` for all current apps.

Reply:
It's done
https://github.com/BrightID/BrightID-Node/commit/375ff351c4f5bb51405527eaa8b352fee3ad8d0b